tpm/key.go

```diff
@@ -185,6 +188,22 @@ func (t *TPM) CreateKey(ctx context.Context, name string, config CreateKeyConfig
 	return
 }
 
+type attestValidationWrapper attest.KeyConfig
+
+func (w attestValidationWrapper) Validate() error {
+	switch w.Algorithm {
+	case "RSA":
+		if w.Size > 2048 {
+			return fmt.Errorf("%d bits RSA keys are (currently) not supported; maximum is 2048", w.Size)
+		}
```

Comment:
ditto

Reply:
Fixed in https://github.com/smallstep/crypto/commit/faa54c3e06ca7e2dfca6fd7af81abfb5f1a0e585.